scripts/unity/func_name_list.py

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/python
+#
+# Copyright (c) 2019 Nordic Semiconductor
+#
+# SPDX-License-Identifier: LicenseRef-BSD-5-Clause-Nordic
+#
+import sys
+import re
+import argparse
+
+def func_names_from_header(in_file, out_file):
```

Comment:
Do you just need the func_names? Not the full prototype?

If so, maybe it would be possible to compile the header and then take the func names from the symbol table.

Then we don't need to regex parse unpreprocessed C code.